packages/shared/routes/setup/Congratulations.svelte

```diff
@@ -117,36 +155,28 @@
 
 <OnboardingLayout allowBack={false}>
     <div slot="leftpane__content">
-        {#if wasMigrated}
-            <div class="relative flex flex-col items-center bg-gray-100 dark:bg-gray-900 rounded-2xl mt-10 p-10 pb-6">
-                <div class="bg-green-500 rounded-2xl absolute -top-6 w-12 h-12 flex items-center justify-center">
-                    <Icon icon="success-check" classes="text-white" />
-                </div>
-                <Text type="h2" classes="mb-6 text-center">{locale('views.congratulations.fundsMigrated')}</Text>
-                <Text type="p" secondary classes="mb-6 text-center">
-                    {locale(`views.congratulations.${localizedBody}`, { values: localizedValues })}
-                </Text>
-                <Text type="h2">{formatUnitBestMatch($totalMigratedBalance, true, 3)}</Text>
-                <Text type="p" highlighted classes="py-1 uppercase">{fiatbalance}</Text>
-            </div>
-        {:else}
-            <div class="relative flex flex-col items-center bg-gray-100 dark:bg-gray-900 rounded-2xl mt-10 p-10 pb-6">
-                <div class="bg-green-500 rounded-2xl absolute -top-6 w-12 h-12 flex items-center justify-center">
-                    <Icon icon="success-check" classes="text-white" />
-                </div>
-                <Text type="h2" classes="mb-5 text-center">{locale('views.congratulations.title')}</Text>
-                <Text type="p" secondary classes="mb-2 text-center"
-                    >{locale(`views.congratulations.${localizedBody}`)}</Text
-                >
+        <div class="relative flex flex-col items-center bg-gray-100 dark:bg-gray-900 rounded-2xl mt-10 p-10 pb-6">
+            <div class="bg-green-500 rounded-2xl absolute -top-6 w-12 h-12 flex items-center justify-center">
+                <Icon icon="success-check" classes="text-white" />
             </div>
-        {/if}
+            <Text type="h2" classes="mb-5 text-center">{locale('views.congratulations.title')}</Text>
+            <Text type="p" secondary classes="mb-2 text-center">{locale('views.congratulations.body')}</Text>
+        </div>
     </div>
-    <div slot="leftpane__action">
+    <div slot="leftpane__action" class="flex flex-col space-y-4">
+        <Button icon="profile" classes="w-full" secondary onClick={migrateAnotherProfile}>
+            {locale('views.congratulations.migrateAnotherProfile')}
+        </Button>
+        {#if isNotALedgerProfile}
+            <Button icon="export" classes="w-full" secondary onClick={handleExportClick}>
+                {locale('views.congratulations.exportStronghold')}
+            </Button>
+        {/if}
         <Button classes="w-full" onClick={() => handleContinueClick()}>
             {locale(`${wasMigrated && !logExported ? 'views.congratulations.exportMigration' : 'actions.finishSetup'}`)}
```

Comment:
For me it doesn't make sense to show the `finish setup` in this case. The user could download their migration log, their stronghold if it is a software profile or migrate another profile. wdyt?